data/mods/DinoMod/monsters/zed-dinosaur.json

```diff
@@ -386,7 +431,7 @@
     "looks_like": "mon_apatosaurus",
     "speed": 50,
     "description": "Massive, long-necked, four-legged dinosaur corpse with a long, whip-like tail.  The head is upright and the neck looks like it would still make a good strong club.",
```

Comment:
```suggestion
    "description": "Massive, long-necked, four-legged dinosaur corpse with a long, whip-like tail.  The head is still upright and the neck looks like it would still make a good, strong club.",
```